File: app/src/main/java/com/example/mymultilanguagedemo/modulejava/demo/MyApplication.java
package com.example.mymultilanguagedemo.modulejava.demo;

import android.app.Activity;
import android.app.Application;
import android.content.res.Configuration;
import android.os.Bundle;

import com.example.mymultilanguagedemo.modulejava.lib.MultiLanguageUtil;


/**
 * Created by lx on 17-10-26.
 */

public class MyApplication extends Application{
    @Override
    public void onCreate() {
        super.onCreate();
        MultiLanguageUtil.init(this);
//        MultiLanguageUtil.getInstance().setConfiguration();
        registerActivityLifecycleCallbacks(new ActivityLifecycleCallbacks() {
            @Override
            public void onActivityCreated(Activity activity, Bundle bundle) {

            }

            @Override
            public void onActivityStarted(Activity activity) {

            }

            @Override
            public void onActivityResumed(Activity activity) {
//                MultiLanguageUtil.getInstance().setConfiguration();
            }

            @Override
            public void onActivityPaused(Activity activity) {

            }

            @Override
            public void onActivityStopped(Activity activity) {

            }

            @Override
            public void onActivitySaveInstanceState(Activity activity, Bundle bundle) {

            }

            @Override
            public void onActivityDestroyed(Activity activity) {

            }
        });
    }

    @Override
    public void onConfigurationChanged(Configuration newConfig) {
        super.onConfigurationChanged(newConfig);
//        MultiLanguageUtil.getInstance().setConfiguration();
    }


}

File: app/src/main/java/com/example/mymultilanguagedemo/kotlinmodule/demo/MyApplication.kt
package com.example.mymultilanguagedemo.kotlinmodule.demo

import android.app.Activity
import android.app.Application
import android.content.res.Configuration
import android.os.Bundle
import com.example.mymultilanguagedemo.kotlinmodule.lib.MultiLanguageUtil

/**
 * Created by lx on 17-10-26.
 */
class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        MultiLanguageUtil.init(this)
        //        MultiLanguageUtil.getInstance().setConfiguration();
        registerActivityLifecycleCallbacks(object : ActivityLifecycleCallbacks {
            override fun onActivityCreated(activity: Activity, bundle: Bundle?) {}
            override fun onActivityStarted(activity: Activity) {}
            override fun onActivityResumed(activity: Activity) {
//                MultiLanguageUtil.getInstance().setConfiguration();
            }

            override fun onActivityPaused(activity: Activity) {}
            override fun onActivityStopped(activity: Activity) {}
            override fun onActivitySaveInstanceState(activity: Activity, bundle: Bundle) {}
            override fun onActivityDestroyed(activity: Activity) {}
        })
    }

    override fun onConfigurationChanged(newConfig: Configuration) {
        super.onConfigurationChanged(newConfig)
        //        MultiLanguageUtil.getInstance().setConfiguration();
    }
}